parsigs/parse_sig_api.py

```diff
@@ -1,12 +1,13 @@
 import sys
-
 import spacy
 from word2number import w2n
 from dataclasses import dataclass
 import re
 from spacy import Language
 from itertools import chain
 import copy
+import os
+from spellchecker import SpellChecker
```

Comment:
something does not make sense, if the dependency is pyspellchecker than the import should be the same